### ISSUE TITLE
Add prefix code to add to all requests

### ISSUE THREAD

**Issue by tristan-ovh** [ovh/ovh-warp10-datasource#48]
Hello,
It is possible to set variables in the warp10 data source that are available in all requests.
But it would be even better to be able to add any code (not just a variable) that would be inserted in all requests. For example I could add one line to raise the quota of number of points that can be fetched.

**Comment by StevenLeRoux** [comment]
Hello Tristan,

Sure! It's called "Execution constants" in the data source and you can put whatever you need in it.

Here is an example on how we use it : 


 - set a token string, reuse it in the scripts using this source

`TOKEN    1234...abcd`

 - set a macro, call it in the script

`MAXOPS <% 'stack.maxops.hard' STACKATTRIBUTE DUP <% ISNULL ! %> <% MAXOPS %> <% DROP %> IFTE %>
`
 - set a secure script, call it in the script

`MYMACRO   <% 'tIf2DfGEegGrewJb5O5LRfa4T3LHL8yUeVPEt4V4nykgn4_iYowmJ1ydGwVItcRVfFP.L34pfRwZ_ecg7LVc0bbJdMWuCEetsf0jnGCBE3(64frer.EZERzETGT789FgNAWdBluPsPhcwCcVa4kJnhkRwhH7yn_802TiwZVTzgZPtghkRgBkpIQt.NHJeEs9zq4LeNwBz9EeezZEffR53F' EVALSECURE %>
`

**Comment by miton18** [comment]
Hi !
There is a tips, the variables you put in the datasource configuration can be a Warp10 macro.
Your execution variable value must starts with a `<%` and ends with a `%>` else it will be interpreted as a string

Example:

Variable `hello` in the datasource
```
<% 'Hello !' %>
```

In your panel query:
```
@hello
```

**Comment by aurrelhebert** [comment]
Yes but this doesn't fix Tristan issue here, as he still have to call the Macro in each of its scripts to be executed :/ 
However it could be a new option in the DataSource? Something like "executed macro"

**Comment by tristan-ovh** [comment]
Indeed using a macro and calling it everywhere could help factorize a bit the code. If it were executed directly, it would be even simpler :)

But thinking again about this problem, for us, having this "script header" in the datasource would not be so great: we have 3 different data sources, and they are in an organization where datasource modification is complicated. The most logical would be to have this code common to all panels of a dashboard, but I guess Grafana doesn't allow that.
